mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -192,6 +196,9 @@ object GBTClassifier extends DefaultParamsReadable[GBTClassifier] {
 
   @Since("2.0.0")
   override def load(path: String): GBTClassifier = super.load(path)
+
+  final val supportedFeatureSubsetStrategies: Array[String] =
```

Comment:
done . I will add this to GBTRegressor in next pull request (forgot to add in this one)